sonar-db/src/main/java/org/sonar/db/qualitygate/ProjectQgateAssociationDao.java

```diff
@@ -36,6 +35,8 @@ public ProjectQgateAssociationDao(MyBatis mybatis) {
     this.mybatis = mybatis;
   }
 
+  @Deprecated
+  // TODO delete
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)